docs/conf.py

```diff
@@ -100,7 +100,10 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = [
+    '_build', 'architecture/copying.md', 'db_dev_process/minitests/index/**',
```

Comment:
One per line would be better.

Reply:
It was like that, but the format auto-check on Travis complained about the formatting.
I pushed again the previous version.